velox/expression/FunctionSignature.cpp

```diff
@@ -229,11 +229,15 @@ SignatureVariable::SignatureVariable(
     std::string name,
     std::optional<std::string> constraint,
     ParameterType type,
-    bool knownTypesOnly)
+    bool knownTypesOnly,
+    bool orderableTypesOnly,
+    bool comaprableTypesOnly)
     : name_{std::move(name)},
       constraint_(constraint.has_value() ? std::move(constraint.value()) : ""),
       type_{type},
-      knownTypesOnly_(knownTypesOnly) {
+      knownTypesOnly_(knownTypesOnly),
```

Comment:
Can we add a check that the new properties apply only when isTypeParameter() true?

Reply:
Added.